assembl/tests/views_tests/test_graphql.py

```diff
@@ -2495,3 +2495,244 @@ def test_query_discussion_homepage(graphql_request, discussion, test_session):
 """
     res = schema.execute(query, context_value=graphql_request)
     assert res.data['discussion']['homepageUrl'] == url
+
+
+def test_query_discussion_langstring_fields(discussion, graphql_request):
+    res = schema.execute(u"""query {
```

Comment:
use the `.graphql` file (see `test_query_text_fields` for example). Same for all new queries/mutations tests